.gitignore

```diff
@@ -1,6 +1,13 @@
 # Temporary Build Files
 build/_output
 build/_test
+# Goland
+.idea
```

Comment:
not 100% sure on getting this in.... I guess... shouldn't those be on your own global `.gitignore` ?

Reply:
You mean by using `git config --global core.excludesfile` option? 

I think not many people actually use that. I didn't hear of that option, nor any of developers in my office (I know that's not representative :wink:)

On top that it is easier for newcomers, so that repo declares those ignores.